doc/man3/SSL_CTX_set_verify.pod

```diff
@@ -109,8 +109,9 @@ may be different.
 
 SSL_CTX_set_verify_depth() and SSL_set_verify_depth() set the limit up
 to which depth certificates in a chain are used during the verification
-procedure. If the certificate chain is longer than allowed, the certificates
-above the limit are ignored. Error messages are generated as if these
+procedure. Neither end-entity nor the trust-anchor certificates count against
+B<depth>. If the certificate chain is longer than B<depth+2>, the certificates
+above the limit are hence ignored. Error messages are generated as if these
```

Comment:
The code in 1.1.0 no longer behaves as though the additional certificates are not present,
instead it correctly returns X509_V_ERR_CERT_CHAIN_TOO_LONG.